actions/actions.py

```diff
@@ -73,9 +89,74 @@ def resume(args):
     assess_status(CONFIGS, args.services)
 
 
+def _update_replicas(server, replicas):
+    balance_required = False
+    path = SWIFT_RINGS[server]
+    if not os.path.exists(path):
+        action_fail("Server {} swift ring file "
+                    "missing from unit.".format(server))
+
+    current_replicas = get_replicas(path)
+    if float(current_replicas) != float(replicas):
```

Comment:
I would suggest doing these conversions inside of a try/except block, and then failing the actions if the parameters don't parse to float.

Reply:
Additionally, you could avoid doing a not comparision here (!=) by having something like -
```
if current_replicas == replicas:
    action_set({server: "Replicas already set to {}".format(replicas)})
else:
    try:
        set_replicas(path, replicas)
```
... etc.